src/ChargeDensityModels/ChargeDensityModels.jl

```diff
@@ -14,16 +14,19 @@ slopes in each dimension.
 struct LinearChargeDensityModel{T <: SSDFloat} <: AbstractChargeDensityModel{T}
     offsets::NTuple{3, T}
     gradients::NTuple{3, T}
+    CS::Symbol
 end
```

Comment:
Use a parametric type instead of an additional field:
`struct LinearChargeDensityModel{T <: SSDFloat, CS <: {:cylindrical, :cartesian}}`

Reply:
Thanks, @lmh91 for the input. I have changed the code accordingly.